### PR TITLE
Package iter.1.3

### PR DESCRIPTION
**Pull request by c-cube**
### `iter.1.3`
Simple abstraction over `iter` functions, intended to iterate efficiently on collections while performing some transformations

---
* Homepage: https://github.com/c-cube/iter/
* Source repo: git+https://github.com/c-cube/iter.git
* Bug tracker: https://github.com/c-cube/iter/issues

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
The tag seems to have changed after the release:
```
[ERROR] The sources of the following couldn't be obtained, aborting:
          - iter.1.3:
            https://github.com/c-cube/sequence/archive/v1.3.tar.gz (Bad checksum, expected md5=2f5e36ae2224f34f3bb0e9a37311d606)
```

**Comment by kit-ty-kate** [comment]
Actually the fact that it was using `https://github.com/c-cube/sequence` instead of `https://github.com/c-cube/sequence` is a bit weird. An opam-publish bug perhaps?

**Comment by kit-ty-kate** [comment]
Nevermind, the same issue arise. So the tag really did change.

**Comment by c-cube** [comment]
Sorry, reposting with newer tag. I had left a prompt unanswered :(

File: packages/iter/iter.1.3/opam [hunk 1]
```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+authors: ["Simon Cruanes" "Gabriel Radanne"]
+maintainer: "simon.cruanes.2007@m4x.org"
+license: "BSD-2-clause"
+synopsis: "Simple abstraction over `iter` functions, intended to iterate efficiently on collections while performing some transformations"
+build: [
+  ["dune" "build" "@install" "-p" name "-j" jobs]
+  ["dune" "build" "@doc" "-p" name "-j" jobs] {with-doc}
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
```
Comment (kit-ty-kate) [inline packages/iter/iter.1.3/opam:9]:
```suggestion
  ["dune" "runtest" "-p" name "-j" jobs] {with-test & arch != "arm32" & arch != "x86_32"}
```
```
#=== ERROR while compiling iter.1.3 ===========================================#
# context              2.1.1 | linux/arm32 | ocaml-base-compiler.4.13.1 | pinned(https://github.com/c-cube/sequence/archive/v1.3.tar.gz)
# path                 ~/.opam/4.13/.opam-switch/build/iter.1.3
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune runtest -p iter -j 159
# exit-code            1
# env-file             ~/.opam/log/iter-2606-c81537.env
# output-file          ~/.opam/log/iter-2606-c81537.out
### output ###
# File "README.md", line 1, characters 0-0:
#          git (internal) (exit 1)
# (cd _build/default && /usr/bin/git --no-pager diff --no-index --color=always -u README.md README.md.corrected)
# diff --git a/README.md b/README.md.corrected
# index 4c8f158..5c59c59 100644
# --- a/README.md
# +++ b/README.md.corrected
# @@ -7,7 +7,7 @@ Clean and efficient loop fusion for all your iterating needs!
#  # #require "iter";;
#  # let p x = x mod 5 = 0 in
#    Iter.(1 -- 5_000 |> filter p |> map (fun x -> x * x) |> fold (+) 0);;
# -- : int = 8345837500
# +- : int = -244097092
#  ```
#  
#  `Iter` is a simple abstraction over `iter` functions
# @@ -95,7 +95,7 @@ use `Iter.(--) : int -> int -> int Iter.t`.
#  
#  ```ocaml
#  # Iter.(1 -- 10_000_000 |> fold (+) 0);;
# -- : int = 50000005000000
# +- : int = 143223616
#  
#  # let p x = x mod 5 = 0 in
#    Iter.(1 -- 5_000
# @@ -103,7 +103,7 @@ use `Iter.(--) : int -> int -> int Iter.t`.
#      |> map (fun x -> x * x)
#      |> fold (+) 0
#    );;
# -- : int = 8345837500
# +- : int = -244097092
#  ```
#  
#  **NOTE**: with _flambda_ under sufficiently strong
#    run_qtest alias qtest/runtest
#  seed: 91532210
# Running tests...l:32                                                  l:32 *                                                l:50                                                  l:50 *                                                l:63                                                  l:63 *                                                l:91                                                  l:91 *                                                l:106                                                 l:106 *                                               l:121                                                 l:121 *                                               l:159                                                 l:159 *                                               l:166                                                 l:166 *                                               l:178                                                 l:178 *                                               l:195                                                 l:195 *                                              l:196                                                l:196 *                                              l:197                                                l:197 *                                              l:221                                                l:221 *                                              l:233                                                l:233 *                                              l:359                                                l:359 *                                              l:367                                                l:367 *                                              l:379                                                l:379 *                                              l:408                                                l:408 *                                              l:427                                                l:427 *                                              l:428                                                l:428 *                                              l:429                                                l:429 *                                              l:449                                                l:449 *                                              l:473                                                l:473 *                                              l:497                                                l:497 *                                              l:516                                                l:516 *                                              l:530                                                l:530 *                                              l:541                                                l:541 *                                              l:558                                                l:558 *                                              l:567                                                l:567 *                                              l:578                                                l:578 *                                              l:666                                                l:666 *                                              l:730                                                l:730 *                                              l:742                                                l:742 *                                              l:773                                                l:773 *                                              l:774                                                l:774 *                                              l:783                                                l:783 *                                              l:799                                                l:799 *                                              l:829                                                l:829 *                                              l:857                                                l:857 *                                              l:867                                                l:867 *                                              l:883                                                l:883 *                                              l:895                                                l:895 *                                              l:914                                                l:914 *                                              l:968                                                l:968 *                                              l:1058                                               l:1058 *                                             l:1111                                               l:1111 *                                             l:1124                                               l:1124 *                                             l:1140                                               l:1140 *                                             l:1141                                               l:1141 *                                             l:1142                                               l:1142 *                                             l:1143                                               l:1143 *                                             l:1147                                               l:1147 *                                             l:1150                                               l:1150 *                                             l:1359                                               l:1359 *                                             l:1473                                               l:1473 *                                             li:398                                               li:398 *                                             li:399                                               li:399 *                                             li:411                                               li:411 *                                             li:421                                               li:421 *                                             li:432                                               li:432 *                                             li:433                                               li:433 *                                             li:359                                         li:359 *                                       li:360                                         li:360 *                                       li:372                                         li:372 *                                       li:382                                         li:382 *                                       li:393                                         li:393 *                                       li:394                                         li:394 *                                       
# SUCCESS
```

**Comment by kit-ty-kate** [comment]
Thanks! Could you return the fix in the opam file upstream (or restrict those specific tests to 64 bit archs in the tests themselves) ?

**Comment by c-cube** [comment]
I did, thank you!

**Pull request merged**